File: backend/src/main/java/top/vuhe/entity/equipment/dto/ThresholdDTO.java
package top.vuhe.entity.equipment.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.RequiredArgsConstructor;

/**
 * @author zhuhe
 */
@Data
@AllArgsConstructor
@RequiredArgsConstructor
public class ThresholdDTO {
    private Integer channel;
    private Integer threshold;
}

File: backend-kt/src/main/kotlin/top/vuhe/sw/entity/equipment/dto/ThresholdDTO.kt
package top.vuhe.sw.entity.equipment.dto

typealias ThresholdDTO = Map<Int, Int>

/**
 * 阈值传送包
 *
 * @property channel   通道号
 * @property threshold 阈值
 */
data class ThresholdPair(
    val channel: Int?,
    val threshold: Int?
)
